model/http-common.yaml

```diff
@@ -94,3 +94,9 @@ groups:
       - ref: url.scheme
         requirement_level: required
         examples: ["http", "https"]
+        note: >
+          The scheme of the original client request, if known
+          (e.g. from [Forwarded](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Forwarded),
+          [X-Forwarded-Proto](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Forwarded-Proto),
+          or a similar header).
+          Otherwise, the scheme of the immediate peer request.
```

Comment:
Thinking about it again, I'm no longer sure this is ideal. From a security perspective, it is now no longer possible to determine where the TLS connection terminates in the trace. But maybe that use case should better be solved by #21 or something like it?